### PR TITLE
Added wrapper over some more parts of API

### PR DESCRIPTION
**Pull request by barnwal-paras**

### CODE REVIEW

**Comment by gvagenas** [comment]
@barnwal-paras lets work on one PR for now. Keep your work on one branch, unless if you have to experiment on something, and keep only one PR open. 

Please update the branches as needed, and close one of the two PRs.

Thanks
George

File: src/AccountManager.cs [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+namespace RestComm
+{
+	
+	public partial class Account
+		{
+		
+		public accountProperties Properties;
+
+
+		public static string baseurl="https://cloud.restcomm.com/restcomm/2012-04-24/";
```
Comment (gvagenas) [inline src/AccountManager.cs]: @barnwal-paras base url should be configurable. The SDK will be used either with cloud.restcomm.com or any other Restcomm-Connect installation

File: src/AccountManager.cs [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+//  */
+//
+using System;
+using System.Collections.Generic;
+using RestSharp;
+using RestSharp.Authenticators;
+
+
+
+namespace RestComm
```
Comment (gvagenas) [inline src/AccountManager.cs]: @barnwal-paras what is the convention for namespace? Can we use `org.restcomm.connect.sdk.dotnet` ?

File: src/AccountManager.cs [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+		public static string baseurl="https://cloud.restcomm.com/restcomm/2012-04-24/";
+
+		//constructor method to instantiate account 
+
+		public Account(string sid,string Tokenno)
+		{
+
+
+			RestClient client = new RestClient(baseurl+"Accounts/"+sid);
```
Comment (gvagenas) [inline src/AccountManager.cs]: @barnwal-paras we should use the JSON endpoints all over the project

**Review by gvagenas — CHANGES_REQUESTED** [review]
@barnwal-paras nice that you separated the unit tests from the actual implementation. 

following some comments for the next steps:

- Please add to gitignore the binaries and objects 
- Make Examples part of the solution, so if one open Restcomm.sln should have Examples also
- Change the solution name to Restcomm-Connect-DotNet-SDK or something similar that will be clear this is the dotnet sdk
- Make the solution a nuget so a developer can include it in his project
- Change the namespace to something that will be clear this is the dotnet sdk, such as `org.restcomm.connect.sdk.dotnet`
- Use the json endpoints
- Base URL must be passed as an argument to the main constructor of the SDK and NOT be hardcoded

The most important of all is the Unit Test should work with a mock of Restcomm-Connect and NOT using the cloud.restcomm.com. Ping me if you need help on that but its very important that you don't test against cloud.restcomm.com

I guess you are working on the following:
- Tutorial
- Documentation

**Comment by gvagenas** [comment]
@barnwal-paras there must be a problem with the solution setup. When I open Restcomm.sln the classes in the src folder doesn't exist.
screen shot 2017-06-22 at 14 39 47

Am I missing something? Can you please check?

Also please open issues for the missing items/tasks. Like this we merge this initial PR and your work from now on will be with PRs based on issues.

**Comment by gvagenas** [comment]
Also provide README.md with the following:

- Give a short description of the project (1-2 lines)
- how to build the project
- how to run the tests
- dotnet version supported
- Reference any dependencies that might be needed to be added by a developer
- Provide instructions for where to download Restcomm and point to the restcomm documentation site
- Provide information how and where developers can open issues and feature requests

Also please provide example application and instructions for how to run it etc.

Imagine you have a dotnet developer but not familiar with Restcomm-Connect, that wants to use the library. Given that, we must provide any necessary information for him to get started.

**Comment by barnwal-paras** [comment]
I am working on documentation and description . Will update that on github by tomorrow .
Can you confirm if you have latest version of the project ( https://github.com/barnwal-paras/restcomm-sdk-dotnet.git ) from branch1 .Project in master branch is not updated .

**Comment by barnwal-paras** [comment]
My bad . Actually i forgot to delete duplicate project file  restcomm.csproj  .please open solution file (Restcomm-dot-net-sdk.sln)  or open Restcomm-dot-net-sdk.csproj inside Restcomm-dot-net-sdk folder to load the solution .

**Comment by gvagenas** [comment]
Ok great, checking now @barnwal-paras

**Comment by gvagenas** [comment]
Great work @barnwal-paras 👍

**Pull request merged**